style.css

```diff
@@ -0,0 +1,342 @@
+html,
+body{
+    width: 1349px;
+    font: 12px/18px 'PT Sans', 'Ubuntu', sans-serif;
+    margin:0;
+    padding: 0;
+}
+
+.container{
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-direction: column;
+    flex-direction: column;
+    -webkit-align-items: flex-start;
+    width: 1349px;
+    height: 820px;
+    margin: 0;
+    padding: 0;
+}
+
+.nav-bar {
+    box-sizing: border-box;
+    max-width: 990px;
+    margin: 80px auto 15px auto;
+    height: 36px;
+    color: #333;
+}
+
+.nav-bar ul {
+    box-sizing: border-box;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    width: 990px;
+    height: 36px;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-direction: row;
+    flex-direction: row;
+}
+
+.nav-items a {
+    box-sizing: border-box;
+    display: block;
+    text-decoration: none;
+    color: #fff;
+    width: 100%;
+    height: 36px;
+}
+
+.nav-items {
+    box-sizing: border-box;
+    width: 11%;
+    height: 36px;
+    float: left;
+    margin: 0;
+    border: 0;
+    background: url(images/nav-passive.png) no-repeat center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
```

Comment:
**[REQUIRED]** The navigation bar should use a [`background -> gradient`](https://www.w3schools.com/cssref/css3_pr_background.asp) for background